src/commands/new.rs

```diff
@@ -52,6 +57,18 @@ impl NewCommand {
                 Some(file) => values_from_file(file.as_path()).await?,
                 None => HashMap::new(),
             };
+            if template.is_some() && self.accept_defaults {
+                let parameters = template.as_ref().unwrap().parameters();
```

Comment:
I know we are checking for `.is_some`, but I would still like to avoid calling `.unwrap()` here if possible.